Cargo.lock

```diff
@@ -858,14 +867,14 @@ version = "0.5.5"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "5444c27eef6923071f7ebcc33e3444508466a76f7a2b93da00ed6e19f30c1ddb"
 dependencies = [
- "windows-sys",
+ "windows-sys 0.48.0",
```

Comment:
I note that the windows-sys crates are quite big (windows-sys itself is 2.5 MiB, and most of the import lib crates it depends on are around 800 KiB each). I think this duplication would probably go away if we updated all of our crates, but it's worth keeping an eye on.

Reply:
I've updated all crates we depend on to their latest releases; we are blocked on de-duplication here by these dependencies having new versions released.